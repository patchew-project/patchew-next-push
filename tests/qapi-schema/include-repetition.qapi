{ 'include': 'comments.qapi' }
{ 'include': 'include-repetition-sub.qapi' }
{ 'include': 'comments.qapi' }
