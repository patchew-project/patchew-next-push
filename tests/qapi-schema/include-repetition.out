object q_empty
enum QType ['none', 'qnull', 'qnum', 'qstring', 'qdict', 'qlist', 'qbool']
    prefix QTYPE
module include-repetition.qapi
include comments.qapi
module comments.qapi
enum Status ['good', 'bad', 'ugly']
module include-repetition.qapi
include include-repetition-sub.qapi
module include-repetition-sub.qapi
include comments.qapi
include comments.qapi
module include-repetition.qapi
include comments.qapi
